tests/src/rules/named.js

```diff
@@ -135,7 +135,6 @@ ruleTester.run('named', rule, {
 
     test({
       code: 'import { a } from "./re-export-names"',
-      args: [2, 'es6-only'],
```

Comment:
note how it's only used here: https://github.com/benmosher/eslint-plugin-import/search?utf8=✓&q=es6-only&type=